lib/s3middleware/azureHelpers/mpuUtils.js

```diff
@@ -188,12 +188,22 @@ log, cb) => {
     hashedStream.on('data', data => {
         const currentLength = streamInterface.getLengthCounter();
         if (currentLength + data.length > azureMpuUtils.maxSubPartSize) {
+            const bytesToMaxSize = azureMpuUtils.maxSubPartSize - currentLength;
+            const firstChunk = bytesToMaxSize === 0 ? data :
+                data.slice(bytesToMaxSize);
```

Comment:
Should this `data.slice` be from 0 - bytesToMaxSize?

Reply:
We write 0 - bytesToMaxSize in the current stream on https://github.com/scality/Arsenal/pull/341/files/53c88cf4c415d0284b490d9ded207aa5f8d0b245#diff-95424c3fe467f0aebd5fb790122f83c1R197
and then write the remainder (from bytesToMaxSize to the end) as the first chunk of the next stream